ChangeLog

```diff
@@ -1,3 +1,79 @@
+        controlled via cluster option priority-fencing-delay (default = 0)
+  + libcrmcommon: Add a formatter for logs
+  + libpe_status: Add several new functions for working with tags.
+  + stonith_admin: add --delay option (default = 0) to support enforced
+                   fencing delay
+  + tools: Add --include= and --exclude= to crm_mon.
+  + tools: Add --node= to filter crm_mon output for a node (or tag).
+  + tools: Add an option for printing failed resource actions.
+  + tools: Add more fine grained fencing arguments.
+  + tools: Allow crm_rule to check some date_specs.
+
+- Changes since Pacemaker-2.0.3
+  + build: improve checking headers for C++ fitness
+  + build: restore build on latest OpenSUSE
+  + gnublib: refresh to e3b56bc2e68f747320d67754df575976266a5712
+  + libstonithd: Report an error getting history in output instead of
+                 empty history
+  + libstonithd: Improve parameter handling for RHCS-style fence-agents
+                 - make parameter `plug` & `action` shown as not required
+		 - add `plug` parameter besides deprecated `port`
+  + controller: clear leaving node's transient attributes even if there is no DC
+  + controller: don't ack same request twice
+  + cts: Add tests for new crm_rule capabilities.
+  + cts: Improve & Correct tests for date_expressions.
+  + executor: correctly convert nanoseconds to milliseconds
+  + executor: handle systemd execution times under 1 second
+  + iso8601: Fix crm_time_parse_offset() to parse offset with plus sign.
+  + libcrmcommon, libpe: validate interval specs correctly
+  + libcrmcommon: Add pcmk_str_is_infinity, pcmk_str_is_minus_infinity and
+                  pcmk__unregister_formats.
+  + libcrmcommon: Extend what pcmk__split_range can do.
+  + libfencing: Export formatted output functions.
+  + libpe_status: Add node list arg to output messages preventing weird
+                  behavior + potential segfaults
+  + libpe_status: Update the maint mode message for HTML output.
+  + libstonithd, tools: Fix arguments to stonith-event.
+  + scheduler: don't incorporate dependencies' colocations multiple times
+  + scheduler: ensure attenuated scores still have some effect
+  + scheduler: ignore colocation constraints with 0 score
+  + scheduler: make sure cluster-wide maintenance-mode=true overrides
+               per-resource settings
+  + scheduler: properly detect whether guest node is fenceable
+  + scheduler: do not differentiate case where all the nodes have equal priority
+  + tests: Add tests for pe_cron_range_satisfied.
+  + tests: Add tests for the current behavior of pcmk__split_range.
+  + tools, lib: Use standard pacemaker return codes in crm_rule.
+  + tools: Correct stonith-event arguments in crm_mon.
+  + tools: Fix man pages for crm_mon & crm_diff.
+  + tools: Make crm_mon --interval understand more formats.
+  + tools: Fix --html-title= in crm_mon.
+  + tools: Fix handling of log verbosity in crm_simulate
```

Comment:
The above entry was only in master (not a release) so it doesn't need to be listed